### DISCUSSION TITLE
UX Kickoff Meeting 10/18/22 - Notes

### DISCUSSION THREAD

**Discussion by Elirudite** [omigroup/ux-research#11] [General]
**Agenda:**

- What our interests are (what do we want to research exactly)
- What resources do we have available (people, skills, time, money, etc)
- What should be our first action item/goal for the group to accomplish

**Introduction - What are our interests with UX research?**
Shadow: a designer in XR. not much coding, but want to help build the metaverse. design work. want to build processes and systems to build interoperable metaverse or metaverse tools. more processes for design in this group

sindhu: not heavily into metaverse, but understands Unity, Unreal,  Blender. Software dev, data architect. Usability oriented. software PM: doing ui research / background. 
Often developed from application perspective: not from what the user wants. Should be developed from the perspective of user

evo: in various forms of virtual worlds, metaverse, second life. cross sector, make tools more accessible to the general public to get more to participate. to make it open, interactive and useful across different verticals. mostly focus on production to the public and strategy side. to make sure people's content is more accessible and so people can use them. helpful to know what is being asked and how you can do it better

Sifr: Mints clothes on the blockchain

Lyuma: From v-sekai group. Passionate about what's accessible to people. Interested in different UI designs 

Praelian: Works in cypersecurity and AI support systems. Interested in how to cheat Fitts Law (ratio between distance target and user)
Intersection between UI and virtual spaces. Figure out new ways to explore UI

Jesse (mrmetaverse): project management experience, business development, rapid prototyping. done UX work with a UX team. product management vs product design (value design). All about iteration, rather than designing first. "just build" culture is the antithesis of what I am, and hope to share practices and discuss things. hope that people can be comfortable coming here and bringing their problems: have users encountered this. avoid drawing rigid lines in the sand)

Jesse (Echoparadigm): product designer and design lead, primarily concerned about qualification of diversity, accessibility within our practices, and furthering such initiatives. interaction design and UI; accessible by many different archetypes, and the spectrum of users who look through different lenses. (e.g. don't have use of an arm; mindful)
Be mindful about accessibility 

Jin - I do a lot of research and building across many platforms, finding good design patterns for things that I really like that make the social experience richer or the builder experience more fun / collaborative. I frequently create and share demos between multiple platforms to devs working on diff platforms and tools, and seem to have a pretty good success rate of ideas being implemented.

Evo - hasn't seen a lot of (or any) UX conversations in various metaverse conferences

Sindhu - XR is not on many usability people's mind

Apple is going to come out soon, we dont want them to define the totality of what makes good UX for XR.

Worried about people who are likely to get swallowed up ... people who don't know any of what is going on 

Coming from the perspective of an individual. 
- What's the day-to-day life of a person going through VR? Going through the internet?

Shadow poses a question: How to make it so it's easier for people who are building these apps to talk to a regular person how they are interacting (picking things up)... how to start this research.

Product management is the alignment of business needs (viability), technical needs (feasibility), and user needs (usability)
- Many businesses approach 

Propose starting with a problem to solve, then go from there.
How to get companies/orgs to bring their problems to us around real people

Sindhu: someone with industry usecases from financial, healthcare, not from metaverse. if I dont' come in with what the individual wants, the developer will do what they want

Sindhu - Have in-depth use cases, feel if people don't have these use cases, then we won't accomplish much.

Mrmetaverse - Want to have orgs bring us their usability problems/opportunities as a more sustainable way for us work

Action steps - Share use cases about problems worth addressing in this group

**Comment by mrmetaverse** [comment]
## OMI - UX Research kickoff 

This group is brand new to OMI. We are going to spend a few sessions learning about ourselves and each other, getting aligned, and getting our repo set up. 


## Attendance / Intros: 
- Fire - Wants to see us afford more processes for design. 
- Sindhu - Software Developer, Data Architect, Project Management. I have a strong UI and UX background. Prefers to see research conducted before code begins. Really wants to see us prioritize what the user wants. 
- Evo - various forms of virtual worlds, and live streaming. Virtual World builds in Second Life, and an early Metaverse roadmap. Works across sectors, government, and civic engagement. Wants to see our work more accessible for the general public. Mostly focused on production back to the public, and strategy. 
- Sifr - creates 3d avatars and mints them on the blockchain using the VRM format. 
- Lyuma - From the V-Sekai group working with Godot engine. We want to make useful tools for the Metaverse, and UX is one of the most important parts. I am passionate about our work being accessible for ordinary people, and not complicated tech stacks that exclude. I have seen a lot of VR games using more 2D UI, and I think what some of the AR people are doing is cool. There is so much opportunity to make something better. 
- Praelian - I work in cybersecurity, and I am interested in how AI support systems can work with VR spaces. Ideally how can we comply with or cheat [Fitts law](https://en.wikipedia.org/wiki/Fitts's_law)? You can touch computers, so what are new ways to explore that aren't immediately rejected by users.
- mrmetaverse - Product Management, business development, workshopping, discovery, and rapid prototyping. I hope this group can become a place to explore and learn together. I like to imagine that builders can bring their challenges to the group and share their experiences with each other. To solve together. 
- echoparadigm - Product Designer, Service Design, Product interaction, and front end design. Design systems. I care about accessibility in our design practices, and hope our work can help ensure the Metaverse is accessible by many different archetypes. 
- Jin 
- Humbletim 
- SM Sith Lord 


## General discussion: 

- There is very little conversation about Metaverse UX at most Metaverse events. 
    - IEEE in VR has done a good job of this. 
- Surprising that its not more popular 
    - we see it at AWE and VRARA
- I think its because the Metaverse is niche right now. I work with usability people all the time, and its not on their radar. I want to make sure it works before the general public really starts getting access. 
- Jesse A: I think when Apple enters the field officially, we will see an explosion of UX discussion in the field. Like the magnitude of the app store launch driving millions to become "app developers". 
- Apple does create some great UI and usability but it forces you into a system. I am tired of applications driving the access and not individuals getting to set their preferences. 
- Sounds like we want to help ensure the Metaverse is more accessible. Not top-down, but more bottom up. We want people to understand how to get involved, and use this technology. 


## What are some of our goals? 
- Drive the process away from app-centric control (Apple walled gardens) and more user centric. 
- Alignment and kickoff, I propose that we review next week 
- How can we encourage outsiders to get involved and bring their UX problems worth solving? 
- Define a early problem statement, and explore various solutions as they relate to specific user personas. 
- Create something that is open and accessible to someone who is 10, or 80. 
- Volunteers to actually do the work is a major bottleneck risk. This has to be sustainable, so I think mutually beneficial problems to solve is the best way to sustain volunteerism and outcomes. If companies and builders can bring us their problems they will have more incentive to be part of the solutioning and discovery. 
- Be a force of change promoting the consideration of usability as it weighs up to feasibility and business requirements (viability). 
- I don't think we should broadly cast a wide net, I think Shadow and Jesse James said it well, we should frame problems worth solving with real personas and incrementally promote change. 



## Next steps: 

Put your ideas in #12 , what are some problems worth solving? 


Our next meeting will be November 1, 2022 at 16:00 UTC. We will create our readme, update our meeting time, complete our first milestone (set up the repo), and last we will review suggestions in #12.

**Reply by mrmetaverse** [reply]
Lyuma - Where should post use cases?

**Reply by mrmetaverse** [reply]
created #12
